wagtail/admin/templates/wagtailadmin/shared/keyboard_shortcuts_dialog.html

```diff
@@ -0,0 +1,30 @@
+{% load wagtailadmin_tags i18n %}
+{% dialog icon_name='regex' id='keyboard-shortcuts-dialog' title=_("Keyboard shortcuts") %}
+    {% if shortcuts%}
```

Comment:
I would not worry about this logic here, as we are currently hard-coding the shortcuts from the template tag there should never be a case where there are no shortcuts.